charts/templates/clustersynchro-manager-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+{{- $name := include "clusterpedia.name" . -}}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $name }}-clustersynchro-manager
+  namespace: {{ include "clusterpedia.namespace" . }}
+  labels:
+  {{- include "clusterpedia.cm.labels" . | nindent 4 }}
+spec:
+  replicas: {{  .Values.clustersynchroManager.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "clusterpedia.cm.labels" . | nindent 6}}
+  template:
+    metadata:
+      {{- with .Values.clustersynchroManager.podAnnotations }}
+      annotations:
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+      {{- include "clusterpedia.cm.labels" . | nindent 8}}
+      {{- include "clusterpedia.cm.podLabels" . | nindent 8}}
+    spec:
+      {{- with .Values.clustersynchroManager.imagePullSecrets }}
+      imagePullSecrets:
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+      - name: {{ $name }}-clustersynchro-manager
+        image: "{{ .Values.clustersynchroManager.image.repository }}:{{ .Values.clustersynchroManager.image.tag | default  "latest" }}"
+        command:
+        - /usr/local/bin/clustersynchro-manager
+        - --storage-config=/etc/clusterpedia/storage/internalstorage-config.yaml
+        - --feature-gates=PruneManagedFields=true,PruneLastAppliedConfiguration=true
```

Comment:
` --leader-elect-resource-namespace` flag needs to be set, otherwise the leader election resource cannot be created becuase of the namespace change.